DEVELOPMENT.md

```diff
@@ -30,61 +43,60 @@ flowchart LR
 ```
 
 During test run, the test suites:
-1. create ephemeral pipelines and queues for a given [Buildkite Agent Cluster](https://buildkite.com/docs/clusters/overview).
-2. Run executor, which will monitor jobs from the target queue in target Buildkite Cluster,
-   starts new Jobs in a Kubernetes cluster.
-3. Test suite will clean up those ephemeral objects in the end.
+1. Create ephemeral pipelines and queues for a given [Buildkite Agent Cluster](https://buildkite.com/docs/clusters/overview).
+2. Run executor, which will monitor jobs from the target queue in target Buildkite Cluster and start new Jobs in a Kubernetes cluster.
```

Comment:
(Comment about the existing document) This should probably refer to the controller, since "executor" is not the name of the stack component.

There's also a missing step or two.

```suggestion
2. Runs the controller, which will monitor jobs from the (just-created) queue in the Buildkite Cluster and start new Jobs in the Kubernetes cluster.
3. Starts a build of the pipeline on Buildkite, which causes Buildkite jobs to become available.
4. Polls Buildkite while waiting for the expected outcome, which may include build success, build failure, and the presence or absence of certain log messages.
```